elements/itemfilter/src/helpers/fetch-value.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Fetch value based on nested key or individual key
+ *
+ * @param {string} key - Key to identify the value
+ * @param {Object} items - Items contain list of key & it's value
+ * @returns {any}
+ */
+export default function fetchValue(key, items) {
```

Comment:
Using the word "fetch" for this function name would imply that something is fetched from e.g. an endpoint; maybe better something like `getValue` or `getNestedValue`?